packages/api/src/graphql/mutation.private.ts

```diff
@@ -599,8 +600,13 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       type: GraphQLNonNull(GraphQLComment),
       args: {input: {type: GraphQLNonNull(GraphQLCommentInput)}},
       async resolve(root, {input}, {authenticate, dbAdapter}) {
+        // TODO: Handle authorization
         const {roles} = authenticate()
-        authorise(CanCreateComment, roles)
+        console.log(roles)
```

Comment:
Do you still need this `console.log` ?

Reply:
I have moved it now from the private mutations to public ones.